src/test/cli/rbd/invalid-snap-usage.t

```diff
@@ -89,6 +89,7 @@
   rbd: snapname specified for a command that doesn't use it
   [22]
   $ rbd bench-write foo@snap
+  rbd: bench-write is deprecated, use rbd bench --io-type write ...
```

Comment:
Sorry, I am not sure I understand your point. do you mean we will see this warning in read case? but I think this only happen in bench-write.